fedmsg.d/example.py

```diff
@@ -17,11 +17,13 @@
         # Make sure the consumer name is correct for your environment
         '/queue/Consumer.msg-client-estuary-updater.env.VirtualTopic.eng.freshmaker.event.state.changed',  # noqa: E501
         '/queue/Consumer.msg-client-estuary-updater.env.VirtualTopic.eng.freshmaker.build.state.changed',  # noqa: E501
-        '/queue/Consumer.msg-client-estuary-updater.env.VirtualTopic.eng.distgit.commit'
+        '/queue/Consumer.msg-client-estuary-updater.env.VirtualTopic.eng.distgit.commit',
+        '/queue/Consumer.msg-client-estuary-updater.env.VirtualTopic.eng.errata.activity.status'
     ],
     'stomp_ssl_crt': '/etc/estuary-updater/consumer.crt',
     'stomp_ssl_key': '/etc/estuary-updater/consumer.key',
     'estuary_updater.neo4j_url': 'bolt://neo4j:neo4j@localhost:7687',
     'estuary_updater.koji_url': 'http://kojihub.domain.com/kojihub',
+    'estuary_updater.errata_api_url': 'https://errata.devel.redhat.com/api/v1/',
```

Comment:
Optional: we've been obscuring the internal URLs in the past (see the line above), so it may be good to continue doing so here for consistency.